xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/LivedataPagination.vue

```diff
@@ -296,6 +296,9 @@ export default {
     },
     showEntryRange() {
       return this.data.meta.pagination.showEntryRange
+    },
+    showPagination() {
+      return (this.logic.getPageCount() > 1) || this.data.meta.pagination.showPaginationOnSinglePage;
```

Comment:
ymmv but the parenthesis as not required here
```suggestion
      return this.logic.getPageCount() > 1 || this.data.meta.pagination.showPaginationOnSinglePage;
```